pkg/operators/cli/clioperator.go

```diff
@@ -58,8 +58,15 @@ const (
 	// output modes for a DataSource in a comma-separated list.
 	AnnotationSupportedOutputModes = "cli.supported-output-modes"
 
+	// AnnotationOutputMode can be used to specify the output mode for a
+	// DataSource.
+	AnnotationOutputMode = "cli.output-mode"
```

Comment:
I'm unsure this annotation is necessary. Maybe we can simply set the `cli.supported-output-modes` as `ModeNone` when we want the clioperator to complete ignore a data source.